Formula/veloren.rb

```diff
@@ -0,0 +1,60 @@
+class Veloren < Formula
+  desc "Multiplayer voxel RPG"
+  homepage "https://veloren.net"
+  url "https://gitlab.com/veloren/veloren.git",
+    tag:      "v0.13.0",
+    revision: "34ae5b9df73f367fa34350337ebba71711f48b86"
+  license "GPL-3.0-or-later"
+
+  depends_on "cmake" => :build
+  depends_on "git-lfs" => :build
+  depends_on "ninja" => :build
+  depends_on "rustup-init" => :build
```

Comment:
This needs to build with the Rust formula. While we do have some formulae that use `rustup` to build, that's just a workaround to get existing formulae working that we don't want to adopt for new formulae.

Reply:
Unfortunately it requires nightly rust, and uses a `rust-toolchain` file to pin a specific nightly version. Is there a way to achieve this without `rustup`?

Reply:
I don't think so.

This seems like a really nice piece of software, but I find it hard to overlook not being able to build with the latest stable version of Rust.

I suggest hosting it in an external tap for now. If it proves to be popular, then it'll show up in our analytics data and that would at least justify including it here.

Reply:
That makes sense, thanks for the feedback! I'll go ahead and close this now.

Reply:
From a quick look at the formula, I think it should be fine. But I'll need to take a closer look at the repo if/when you decide to open a PR.